### DISCUSSION TITLE
HTTPS and SSH not working

### DISCUSSION THREAD

**Discussion by adrienrichmond** [OnionUI/Onion#1553] [Support]
Hello, I'm trying to get HTTP and SSH to work on my Miyoo and I'm enabling it. I can't access the HTTP dashboard, SSH doesn't work either. I'm at Free. Thanks in advance

**Comment by oherau** [comment]
First you can:
1-check your wifi config is OK on the device
2-your device has an IP on your local network
3-your services are activated on the device
You should then access to your device from a terminal using this command:
`ssh onion@[ip-of-your-device]  (password: onion)`

For more detail you can refer to this page:
[https://onionui.github.io/docs/network/ssh](https://onionui.github.io/docs/network/ssh)

Tell us more at where step you are stuck so we can help you
